app/components/gh-psm-visibility-input.js

```diff
@@ -8,9 +10,15 @@ const VISIBILITIES = [
 
 export default Component.extend({
 
+    settings: service(),
+
     // public attrs
     post: null,
 
+    selectedVisibility: computed('post', function () {
+        return this.post.get('visibility') || this.settings.get('defaultContentVisibility');
```

Comment:
@kevinansfield wdyt about such approach to setting the default when PSM is opened first thing when creating a new post? Is there a cleaner way?